### PR TITLE
Update 1/10/2024

### PR DESCRIPTION
**Pull request by Alando1-doom**
Updates include:

- Updated KEYCONF; Reorganized some stuff.
- Updated SNDINFO for the accessible menu
- Added new menu sounds
- Updated zscript lump
- Updated Area Scanner Sound Bindings; added pile of skull and candles.
- Updated Menu Sound Bindings

### CODE REVIEW

**Comment by proydoha** [comment]
I see that you're adding a new KEYCONF.txt file and the old one is still in place. This can't be right.

**Review by proydoha — CHANGES_REQUESTED** [review]
Please double check your KEYCONF files. You have two right now: KEYCONF and KEYCONF.txt

image

File: zscript.zs [hunk 1]
```diff
@@ -3,10 +3,14 @@ version "4.6"
 //Sound queue utils
 #include "zscript/Utils/SoundQueue/Toby_SoundQueueStaticHandler.zs"
 #include "zscript/Utils/SoundQueue/Toby_QueuedSound.zs"
+#include "zscript/Utils/SoundQueue/Toby_NumberToVoice.zs"
+#include "zscript/Utils/SoundQueue/Toby_StringToVoice.zs"
 #include "zscript/Utils/SoundQueue/Toby_StringToSoundQueue.zs"
+#include "zscript/Utils/SoundQueue/Toby_OrdinalToVoice.zs"
+#include "zscript/Utils/SoundQueue/Toby_MonthToVoice.zs"
```
Comment (proydoha) [inline zscript.zs:10]: I've removed those classes:
Toby_NumberToVoice
Toby_StringToVoice
Toby_OrdinalToVoice
Toby_MonthToVoice

Are you sure you've pulled latest changes before doing this?

Comment (Alando1-doom) [reply zscript.zs:10]: I double checked and I did, however, some stuff might have crossed over while compiling things into the pk3 file. I will make the corrections on my end. Sorry for the mishap.

**Pull request merged**